packages/charts/react-charting/src/components/VerticalBarChart/VerticalBarChart.base.tsx

```diff
@@ -118,8 +119,8 @@ export class VerticalBarChartBase
       dataForHoverCard: 0,
       isCalloutVisible: false,
       refSelected: null,
-      selectedLegend: props.legendProps?.selectedLegend ?? '',
-      activeLegend: '',
+      selectedLegends: [],
```

Comment:
>selectedLegends [](http://example.com/codeflow?start=6&length=15)

Could you take the change to set selectedLegends to props in componentDidUpdate as in this commit.
[https://github.com/microsoft/fluentui/commit/6a70a11ddb8bac592c4c9b4c12a2705a9ace6686](https://github.com/microsoft/fluentui/commit/6a70a11ddb8bac592c4c9b4c12a2705a9ace6686)

Also update the legendprops to enablemultiselect and selectedlegends value in declarative chart.tsx.

Same for VSBC and GVBC.